docs/includes/installing-fastlane.md

```diff
@@ -1,28 +1,47 @@
-_fastlane_ can be installed multiple ways. The preferred method is with a `Gemfile`. _fastlane_ can also get installed directly through RubyGems or with Homebrew (if on macOS).
+_fastlane_ can be installed multiple ways. The preferred method is with [_Bundler_](https://bundler.io). _fastlane_ can also be installed directly through with Homebrew (if on macOS). Technically, you can use macOS's system Ruby but it's not recommended unless you are a experienced Ruby developer.
 
-#### A Gemfile (macOS/Linux/Windows)
+#### Managed Ruby environment + Bundler (macOS/Linux/Windows)
 
-It is recommended that you use a `Gemfile` to define your dependency on _fastlane_. This will clearly define the used _fastlane_ version, and its dependencies, and will also speed up using _fastlane_.
+**Ruby**
 
+If you use macOS, please don't use system Ruby when possible. [There is a variety of ways to install Ruby without having to modify your system environment](https://www.ruby-lang.org/en/documentation/installation/#managers). For macOS and Linux, _rbenv_ is one of the most popular ways to manage your Ruby environment.
+
+_fastlane_ supports Ruby versions 2.4 through 2.7. Verify which Ruby version you're using:
+
+```sh
+$ ruby --version
+ruby 2.7.2p137 (2020-10-01 revision 5445e04352) [x86_64-darwin19]
+```
+
+**Bundler**
+
+It is recommended that you use _Bundler_ and `Gemfile` to define your dependency on _fastlane_. This will clearly define the  _fastlane_ version to be used and its dependencies, and will also speed up _fastlane_ execution.
```

Comment:
```suggestion
It is recommended that you use _Bundler_ and `Gemfile` to define your dependency on _fastlane_. This will clearly define the _fastlane_ version to be used and its dependencies, and will also speed up _fastlane_ execution.
```